mitiq/calibration/calibrator.py

```diff
@@ -210,12 +222,22 @@ def run(self, log: bool = False) -> None:
                 self.cirq_executor, bitstring_to_measure
             )
             noisy_value = expval_executor.evaluate(circuit)[0]
-            for strategy in self.strategies:
+            for count, strategy in enumerate(self.strategies):
                 with warnings.catch_warnings():
                     warnings.simplefilter("ignore", UserWarning)
                     mitigated_value = strategy.mitigation_function(
                         circuit, expval_executor
                     )
+                if (
+                    log
+                    and count
+                    and strategy.technique
+                    is not self.strategies[count - 1].technique
+                ):
```

Comment:
I think for readability the last condition
```py
strategy.technique is not self.strategies[count - 1].techique
```
should be in parenthesis. Technically, I think it's not totally needed based on pythons [operator precedence](https://docs.python.org/3/reference/expressions.html#operator-precedence) (`is not` binds more tightly than `and` if I'm reading this correctly), but IMO it reads a little easier. Not totally necessary, but WDYT?

Reply:
That makes sense, I've added the parentheses.